### PR TITLE
Feature/sims 1540 new m5

### PR DESCRIPTION
**Pull request by rhiannonlynne**
New m5 values. New FWHMeff + FWHMgeom.

### CODE REVIEW

File: tools/schema_tools/gen_output.py [hunk 1]
```diff
@@ -27,27 +27,34 @@ def getDbData(cursor, sql):
...
-            'y1':0.11}
-    tauCloud = 0.;
-    # Using Kem's SkyBrightness
-    m5 = (Cm[visitFilter] + 0.50*(filtsky-21) + 2.5*math.log10(0.7/seeing) +
-          1.25*math.log10(expTime/30) - kAtm[visitFilter]*(airmass-1) + 1.1*tauCloud)
+            'z':0.05,
+            'y':0.17}
+    # Calculate adjustment if readnoise is significant for exposure time
+    # (see overview paper, equation 7)
+    T = expTime / 30.0
```
Comment (mareuter) [inline tools/schema_tools/gen_output.py]: You defined this but never used it. Was this meant for the m5 formula?

**Comment by mareuter** [comment]
This looks fine code wise except for note. It runs fine in my testing. I can't verify the equation since I don't have the write up.

**Comment by rhiannonlynne** [comment]
I did some updates to the Cm and kAtm values, in response to some updated info from @ivezic. 
(also tagging @ivezic so he can make a comment about the equations). 

The T value was actually used, but clearly T was not a great variable name as it is too easy to not notice. Changed to Tscale (and actually used it in m5 calculation, not just dM5 piece).

**Comment by ivezic** [comment]
Comment about equations: they are beautiful! And probably correct.

**Comment by mareuter** [comment]
I think that's a vote of confidence from everyone. You are good to merge this.

**Pull request merged**